src/SocialClient.ts

```diff
@@ -44,6 +51,7 @@ export class SocialClient implements SocialAPI {
         // Start the client
         await matrixClient.startClient({
             pendingEventOrdering: 'detached', // Necessary for the SDK to work
+            disablePresence,
             initialSyncLimit: 20, // This is the value that the Matrix React SDK uses
         });
```

Comment:
What about this instead

```suggestion
        await matrixClient.startClient({
            pendingEventOrdering: 'detached', // Necessary for the SDK to work
            initialSyncLimit: 20, // This is the value that the Matrix React SDK uses
            ...options
        });
```